.gitignore

```diff
@@ -0,0 +1,2 @@
+*.pyc
+/db.sqlite3
```

Comment:
We should use postgres for our database. Even if this is only for testing, the two databases have differences that may complicate testing and reproducability. Is it possible to have something we can run using Docker or Vagrant to allow for easy local development? I can go over what we did for LORE and the teacher's portal if it's helpful